index.bs

```diff
@@ -407,6 +417,24 @@ spec:url; type:dfn; text:scheme
 
     1. If the [=scheme=] of |url| is not an [=HTTP(S) scheme=], then abort these steps.
 
+        <div class="issue">
+          Would it be more consistent to treat this like an invalid URL, and close?
+        </div>
```

Comment:
Making the failure case (including for being in a nested BC) uniformly close seems reasonable to me. Right now there are some cases where the algorithm returns early but doesn't close, which seems like an unnecessary complexity for the author's mental state machine.

Reply:
ok